lib/src/runner/reporter/xunit.dart

```diff
@@ -0,0 +1,353 @@
+// Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library test.runner.reporter.xunit;
+
+import 'dart:async';
+
+import '../../backend/live_test.dart';
+import '../../backend/state.dart';
+import '../../backend/group.dart';
+import '../../frontend/expect.dart';
+import '../../utils.dart';
+import '../engine.dart';
+import '../load_suite.dart';
+import '../reporter.dart';
+
+class XunitFailureResult {
+  String name;
+  String stack;
```

Comment:
it ends up being a string due to this, https://github.com/jayudey-wf/test/pull/1/files#diff-a649c4838eb8bcc2481aaefa44dec124R205